docs/install-and-run.md

```diff
@@ -121,86 +131,88 @@ In cases where CocoaPods is not compatible with your project, you can add EarlGr
 
 #### Step 1: Generate EarlGrey.framework
 
-  1. Download the source for EarlGrey and its dependencies.
-     * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/archive/1.0.0.zip)
+1. Download the source for EarlGrey and its dependencies.
+  * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/releases/latest)
 
-     * Unzip and go to the `EarlGrey-1.0.0/EarlGrey` directory that contains **EarlGrey.xcodeproj**.
+  * Unzip and go to the `EarlGrey-x.y.z/EarlGrey` directory that contains **EarlGrey.xcodeproj**.
```

Comment:
This should be 2: Unzip and go to the EarlGrey-x.y.z/EarlGrey directory that contains EarlGrey.xcodeproj.

This should be 3: Open the EarlGrey.xcodeproj file

This should be step 4: Build the EarlGrey scheme with note:

As part of the initial build step, a script [**setup-earlgrey.sh**] ....